roles/openshift_web_console/tasks/install.yml

```diff
@@ -73,6 +83,29 @@
         value: "{{ openshift.master.public_api_url }}"
     separator: '#'
     state: present
+  when: config_to_migrate.assetConfig is not defined
+
+- name: Migrate asset config from master-config.yaml
+  yedit:
+    src: "{{ mktemp.stdout }}/{{ __console_config_file }}"
+    edits:
+    - key: clusterInfo#consolePublicURL
+      value: "{{ config_to_migrate.assetConfig.publicURL }}"
+    - key: clusterInfo#masterPublicURL
+      value: "{{ config_to_migrate.assetConfig.masterPublicURL }}"
+    - key: clusterInfo#logoutPublicURL
+      value: "{{ config_to_migrate.assetConfig.logoutURL | default('') }}"
+    - key: clusterInfo#metricsPublicURL
+      value: "{{ config_to_migrate.assetConfig.metricsPublicURL | default('') }}"
+    - key: clusterInfo#loggingPublicURL
+      value: "{{ config_to_migrate.assetConfig.loggingPublicURL | default('') }}"
+    - key: servingInfo#maxRequestsInFlight
+      value: "{{ config_to_migrate.assetConfig.servingInfo.maxRequestsInFlight | default(0) }}"
+    - key: servingInfo#requestTimeoutSeconds
+      value: "{{ config_to_migrate.assetConfig.servingInfo.requestTimeoutSeconds | default(0) }}"
```

Comment:
Is there anything else from `assetConfig.servingInfo` that is safe to migrate? It looked like bind address and the certificates wouldn't work now that we're running in a pod.